Tests/SwiftGenKitTests/StringParserTests.swift

```diff
@@ -50,6 +50,15 @@ class StringParserTests: XCTestCase {
     XCTAssertEqual(placeholders, [.unknown, .int, .object, .float, .char])
   }
 
+  func testParseFlags() throws {
+    let formats = ["%-d", "%+d", "% d", "%#d", "%0d"]
+
+    for format in formats {
+      let placeholders = try Strings.PlaceholderType.placeholders(fromFormat: format)
+      XCTAssertEqual(placeholders, [.int], "Failed to parse format \"\(format)\"")
+    }
+  }
+
```

Comment:
Might be also worth a reverse-test, like ensure that `Strings.PlaceholderType.placeholders(fromFormat: "%_d")` does NOT lead to `[.int]`.